src/Model/Table/QueuedJobsTable.php

```diff
@@ -125,6 +125,16 @@ public function initialize(array $config): void {
 		]);
 	}
 
+	/**
+	 * @return \Cake\Database\Schema\TableSchemaInterface
+	 */
+	public function getSchema(): TableSchemaInterface {
+		$schema = parent::getSchema();
+		$schema->setColumnType('data', 'json');
```

Comment:
thinking about it - I personally would put that in the `initialize()` method via
```
$this->getSchema()->setColumnType('data', 'json');
```

Reply:
What is the official way?^^ I wonder

Reply:
at least for the EnumType we recommend doing it this way.
https://book.cakephp.org/5/en/orm/database-basics.html#enum-type